packages/component-library-react/src/FormFieldTextbox.test.tsx

```diff
@@ -0,0 +1,881 @@
+    name: 'subject',
+    label: 'Subject',
+  };
+
+  it('renders an HTML div element', () => {
+    const { container } = render(<FormFieldTextbox {...defaultProps} />);
+
+    const field = container.querySelector('div');
+
+    expect(field).toBeInTheDocument();
+  });
+
+  it('renders a design system BEM class name: utrecht-form-field', () => {
+    const { container } = render(<FormFieldTextbox {...defaultProps} />);
+
+    const field = container.querySelector('div');
+
+    expect(field).toHaveClass('utrecht-form-field');
+  });
+
+  it('displays as CSS block element (or equivalent)', () => {
+    const { container } = render(<FormFieldTextbox {...defaultProps} />);
+
+    const field = container.querySelector('div');
+
+    expect(field).toBeVisible();
+    expect(field).not.toHaveStyle({ display: 'inline' });
+    expect(field).not.toHaveStyle({ display: 'inline-block' });
+  });
+
+  it('renders rich text content', () => {
+    const { container } = render(
+      <FormFieldTextbox {...defaultProps}>
+        <hr />
+      </FormFieldTextbox>,
+    );
+
+    const richText = container.querySelector('hr');
+
+    expect(richText).toBeInTheDocument();
+  });
+
+  it('can be hidden', () => {
+    const { container } = render(<FormFieldTextbox {...defaultProps} hidden />);
+
+    const field = container.querySelector('div');
+
+    expect(field).not.toBeVisible();
+  });
+
+  it('can have a custom class name', () => {
+    const { container } = render(<FormFieldTextbox {...defaultProps} className="invalid" />);
+
+    const field = container.querySelector('div');
+
+    expect(field).toHaveClass('invalid');
+  });
+
+  it('can have a additional class name', () => {
+    const { container } = render(<FormFieldTextbox {...defaultProps} className="large" />);
+
+    const field = container.querySelector(':only-child');
+
+    expect(field).toHaveClass('large');
+    expect(field).toHaveClass('utrecht-form-field');
+  });
+
+  describe('label', () => {
+    it('renders a design system BEM class name: utrecht-form-field__label', () => {
+      const { container } = render(<FormFieldTextbox {...defaultProps} />);
+
+      const field = container.querySelector('.utrecht-form-field__label');
+
+      expect(field).toBeInTheDocument();
+    });
+
+    it('renders rich text content', () => {
+      const { container } = render(
+        <FormFieldTextbox
+          {...defaultProps}
+          label={
+            <>
+              I can speak the <i lang="la">lingua franca</i>
+            </>
+          }
+        />,
+      );
+
+      const richText = container.querySelector('i');
+
+      expect(richText).toBeInTheDocument();
+    });
+
+    it('is associated with the textbox', () => {
+      render(<FormFieldTextbox {...defaultProps} />);
+
+      const textbox = screen.getByRole('textbox', { name: 'Subject' });
+
+      expect(textbox).toBeInTheDocument();
+    });
+
+    it.skip('can be clicked to focus the textbox', () => {
```

Comment:
Maybe add a comment why this test is skipped for now, so we know when to un-skip it in the future?